torchsde/_core/methods/log_ode.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Log-ODE scheme constructed by combining Lie-Trotter splitting with the explicit midpoint method.
+
+The scheme uses Levy area approximations.
+"""
+
+from .. import adjoint_sde
+from .. import base_solver
+from ...settings import SDE_TYPES, NOISE_TYPES, LEVY_AREA_APPROXIMATIONS
+
+
+class LogODEMidpoint(base_solver.BaseSDESolver):
+    strong_order = 0.5
```

Comment:
This should get strong order 1.0 for every noise, I think.

Reply:
I'm not too sure about that. Is Foster accurate enough?